modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -50,9 +50,9 @@ After aggregation, the variables are:
 
 == Error Handling Inside Scatter-Gather Routes
 
-You can use a Try scope in each route of a Scatter-Gather component to handle any errors that might be generated by a route’s event processors. After every route executes, if any has failed with an error, then the Scatter-Gather component throws an error of type `MULE:COMPOSITE_ERROR`, and event processing does not proceed past the Scatter-Gather component in the flow. Instead, the flow branches to your error-handling event processors.
+You can use a Try scope in each route of a Scatter-Gather component to handle any errors that might be generated by a route’s event processors. After every route executes, if any has failed with an error, then the Scatter-Gather component throws an error of type `MULE:COMPOSITE_ROUTING`, and event processing does not proceed past the Scatter-Gather component in the flow. Instead, the flow branches to your error-handling event processors.
```

Comment:
MULE:COMPOSITE_ERROR causes an exception: org.mule.runtime.core.api.config.ConfigurationException: Could not find ErrorType for the given identifier: 'MULE:COMPOSITE_ERROR'

But should the above fix be MULE:COMPOSITE_ROUTING or perhaps just COMPOSITE_ERROR, as shown in the example below?

Reply:
`MULE:COMPOSITE_ROUTING` or `COMPOSITE_ROUTING` are both okey